src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -1911,7 +1928,7 @@ private void onDecodeError(
         {
             cleanupStreams(traceId, authorization);
             doEncodeClose(traceId, authorization, errorType, errorDescription);
-            doNetworkEnd(traceId, authorization);
+            doNetworkEndIfNecessary(traceId, authorization);
```

Comment:
If network end is conditional, then don't we need to also guard against encoding `CLOSE` after already ended?